dev-support/Jenkinsfile

```diff
@@ -57,7 +56,12 @@ pipeline {
             }
         }
 
-        stage ('precommit-run') {
+        stage ('precommit-run Debian 10') {
```

Comment:
Can we add comments explaining why this stage was added and when it would run?

Reply:
Done.